src/Port.js

```diff
@@ -0,0 +1,14 @@
+class Port {
+  constructor(portName){
+    this.portName = portName;
+    this.ships = [];
+  };
+  addShip(ship) {
+    this.ships.push(ship);
+  };
+  removeShip(ship) {
+    this.ships.pop(ship);
```

Comment:
Is the ship to be removed always the last element in the array? If not it would be more secure to search over the list and only remove a specific one